.env.example

```diff
@@ -10,6 +10,22 @@ CUBEJS_DB_TYPE=postgres
 CUBEJS_API_SECRET= # random 32-digit base64 string - generate from https://generate.plus/en/base64
 CUBEJS_DEV_MODE=true
 
+PORT=4000
+
 # local web server
-APP_API_PORT=4000
 APP_DASHBOARD_PORT=3000
+
+# redis configuration
+CUBEJS_REDIS_URL='redis://host.docker.internal:6379'
+CUBEJS_REDIS_PASSWORD=
```

Comment:
is this the right default?

Reply:
yep. by default redis docker doesn't come with a password. Elasticache also doesn't come with a password by default. However Elasticache redis connection can only be made within the same VPC.